### PR TITLE
NazarYakh - buyProductTest

### PR DESCRIPTION
**Pull request by NazarYakh**

### CODE REVIEW

**Review by NyxLex — APPROVED** [review]

File: src/main/java/com/ss/ita/rozetka/pageobject/pages/OrderingPage.java [hunk 1]
```diff
@@ -27,4 +28,51 @@ public boolean isOrderingPageOpened() {
...
+    public OrderingPage setRequiredFields(String surname, String name, String number) {
+        setSurname(surname);
+        setName(name);
+        setPhoneNumber(number);
+        return this;
+    }
+
+    @Step("OrderingPage: get name error message")
+    public boolean getNameErrorMessage() {
+        return $x("//p[contains(text(),\"Введіть своє ім'я кирилицею\")]").isDisplayed();
```
Comment (toleksyn) [inline src/main/java/com/ss/ita/rozetka/pageobject/pages/OrderingPage.java:60]: do not use Cyrillic in locators
same for all

File: src/main/java/com/ss/ita/rozetka/pageobject/pages/OrderingPage.java [hunk 2]
```diff
@@ -27,4 +28,51 @@ public boolean isOrderingPageOpened() {
...
+        return $x("//p[contains(text(),'Введіть своє прізвище кирилицею')]").isDisplayed();
+    }
+
+    @Step("OrderingPage: get phone number  error message")
+    public boolean getPhoneNumberErrorMessage() {
+        return $x("//p[contains(text(),'Введіть номер мобільного телефону')]").isDisplayed();
+    }
+
+    @Step("OrderingPage: confirm the order")
+    public void confirmTheOrder() {
```
Comment (toleksyn) [inline src/main/java/com/ss/ita/rozetka/pageobject/pages/OrderingPage.java:74]: confirmOrdering

Comment (toleksyn) [reply src/main/java/com/ss/ita/rozetka/pageobject/pages/OrderingPage.java:74]: PO method should return this or new PO

File: src/test/java/com/ss/ita/rozetka/test/BuyProductTest.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+        String urlBeforeConfirmingOrder = getCurrentUrl();
+
+        orderingPage.confirmTheOrder();
+
+        String urlAfterConfirmingOrder = getCurrentUrl();
+
+        assertThat(urlBeforeConfirmingOrder)
+                .as("Url should be different")
+                .isEqualTo(urlAfterConfirmingOrder);
```
Comment (toleksyn) [inline src/test/java/com/ss/ita/rozetka/test/BuyProductTest.java:59]: different or same?

**Review by toleksyn — CHANGES_REQUESTED** [review]

**Pull request merged**